js/map.getStyles.js

```diff
@@ -0,0 +1,85 @@
+//FirefoxSprint isn't here. Look for ffQSprint
+var ffQSprint = {
+	getStyles: function (mapStyles) {
+    return [
```

Comment:
please indent this correctly, this should be a level further in. Also note that we are using 2 spaces per level.